src/03_modules.py

```diff
@@ -6,26 +6,38 @@
 """
 
 import sys
+
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+print(sys.argv)
 
 # Print out the OS platform you're using:
 # YOUR CODE HERE
 
+print(sys.platform)
+
 # Print out the version of Python you're using:
 # YOUR CODE HERE
-
+print(sys.version)
 
 import os
+
 # See the docs for the OS module: https://docs.python.org/3.7/library/os.html
 
 # Print the current process ID
 # YOUR CODE HERE
 
+print(os.getpid())
+
 # Print the current working directory (cwd):
 # YOUR CODE HERE
+print(os.getcwd())
 
 # Print out your machine's login name
 # YOUR CODE HERE
+
+import getpass
+
+print(getpass.getuser())
```

Comment:
Instead of importing `getpass` I would use the `getlogin` method from `os` like so..

```
print(os.getlogin())
```